snapd.go

```diff
@@ -60,6 +60,14 @@ var (
 		Usage:  "Path for logs. Empty path logs to stdout.",
 		EnvVar: "SNAP_LOG_PATH",
 	}
+	flLogTruncate = cli.BoolFlag{
+		Name:  "log-truncate",
+		Usage: "Log file truncating mode. Default is false => append (true => truncate).",
+	}
+	flLogColors = cli.BoolTFlag{
+		Name:  "log-colors",
+		Usage: "Log file coloring mode. Default is true => colored (--log-colors=false => no colors).",
```

Comment:
Seems like it is not possible with the cli library to specify the Value: for booleans.
BoolFlag => false by default
BoolTFlab => true by default

Please note also that the usage implied by this cli library is not symetrical:
BoolFlag log-truncate => false by default => --log-truncate sets it to true
I would have expected the behaviour to be
BoolTFlag log-colors => true by default => --no-log-colors sets it to false
However the way to get it working is by using --log-colors=false :-(